engine/src/test/java/com/redhat/cloud/notifications/processors/drawer/DrawerProcessorTest.java

```diff
@@ -116,6 +145,14 @@ void shouldCreateTwoDrawerNotifications() {
         assertNotNull(event);
         assertEquals(createdEvent.getRenderedDrawerNotification(), event.getRenderedDrawerNotification());
 
+        await().until(() -> inMemorySink.received().size() == 2);
+        Message<String> message = inMemorySink.received().get(0);
+        assertNotNull(message);
+        CloudEventMetadata cloudEventMetadata = message.getMetadata(CloudEventMetadata.class).get();
+
+        Log.info(cloudEventMetadata.getId());
+        Log.info(cloudEventMetadata.getType());
+        Log.info(message.getPayload());
```

Comment:
Did you add log entries because assertions didn't work here?

Reply:
You are right it was for debug purpose :)